src/views/open-api-docs-view/utils/get-operation-props.ts

```diff
@@ -59,6 +63,11 @@ export function getOperationProps(
 			operationObjects.push({
 				operationId: operation.operationId,
 				slug: slugify(snakeCase(operation.operationId), { lower: true }),
+				type,
+				path: {
+					full: path,
+					truncated: truncateHcpOperationPath(path),
+				},
```

Comment:
@zchsh Wasn't sure here how you wanted to handle adding in the props. I figured it'd be nice to align the component props with the `OperationProps`. Let me know!